imjoy/runners/subprocess.py

```diff
@@ -12,29 +12,24 @@
 
 import GPUtil
 
+from imjoy import __version__, API_VERSION, HERE
 from imjoy.connection.decorator import socketio_handler as sio_on
-from imjoy.const import (
-    API_VERSION,
-    DEFAULT_REQUIREMENTS_PY2,
-    DEFAULT_REQUIREMENTS_PY3,
-    NAME_SPACE,
-    REQ_PSUTIL,
-    REQ_PSUTIL_CONDA,
-    TEMPLATE_SCRIPT,
-    __version__,
-)
-from imjoy.helper import (
+from imjoy.connection.server import NAME_SPACE
```

Comment:
This is a bit better though since it's imported from the connection, which is a central feature.

Reply:
Right, I think I can simply remove NAME_SPACE, because that's the default value: https://github.com/miguelgrinberg/python-socketio/blob/master/socketio/namespace.py#L3